src/main/java/seedu/address/model/session/Duration.java

```diff
@@ -25,14 +25,33 @@ public Duration(String value) {
         this.value = value;
     }
 
+    public String getValue() {
+        return value;
```

Comment:
I believe `this.value` would be better? 🤡

Reply:
Amended in next commit!